ibm/resource_ibm_cis_certificate_order.go

```diff
@@ -0,0 +1,198 @@
+package ibm
+
+import (
+	"log"
+	"time"
+
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	cisCertificateOrderID            = "certificate_id"
+	cisCertificateOrderHosts         = "hosts"
+	cisCertificateOrderType          = "type"
+	cisCertificateOrderTypeDedicated = "dedicated"
```

Comment:
why is this not added in schema..
In future there might be a possibility of different types right?

Reply:
answered in webex